cargo-dist/src/backend/installer/homebrew.rs

```diff
@@ -26,6 +26,8 @@ pub struct HomebrewInstallerInfo {
     pub homepage: Option<String>,
     /// A brief description of the application
     pub desc: Option<String>,
+    /// A GitHub repository to write the formula to, in owner/name format
+    pub tap: Option<String>,
```

Comment:
Does homebrew only support github taps, or is there also an extended format we should (one day) support?

Reply:
The default `brew tap` and `brew install` syntaxes uses GitHub, but you can use non-GitHub git repos by specifying a full URL to the `brew tap` command.